BUILDING.md

```diff
@@ -175,7 +169,7 @@ Binaries at <https://nodejs.org/download/release/> are produced on:
 | linux-ppc64le           | RHEL 8 with gcc-toolset-10[^6]                                                                              |
 | linux-s390x             | RHEL 8 with gcc-toolset-10[^6]                                                                              |
 | linux-x64               | RHEL 8 with gcc-toolset-10[^6]                                                                              |
-| win-x64 and win-x86     | Windows Server 2022 (x64) with Visual Studio 2022                                                           |
+| win-x64                 | Windows Server 2022 (x64) with Visual Studio 2022                                                           |
```

Comment:
Doesn't have to be in this PR, but we should probably have an entry in the table for win-arm64.